src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -101,11 +100,66 @@ internal WebApplicationBuilder(Assembly? callingAssembly, string[]? args = null)
         /// <returns>A configured <see cref="WebApplication"/>.</returns>
         public WebApplication Build()
         {
+            // Copy the configuration sources into the final IConfigurationBuilder
+            _hostBuilder.ConfigureHostConfiguration(builder =>
```

Comment:
Does it matter this was moved out of ConfigureWebHost? Is there something specific these callbacks need to run before?

Reply:
I just don't see why it needs to run inside of ConfigureWebHost

Reply:
Fair.